content/docs/tutorials/analyze_pages_and_fix_errors.md

```diff
@@ -109,29 +109,36 @@ Snapshots can tell you if:
 
 In Apify scrapers (**Web Scraper** ([apify/web-scraper](https://apify.com/apify/web-scraper)), **Cheerio Scraper** ([apify/cheerio-scraper](https://apify.com/apify/cheerio-scraper)) and **Puppeteer Scraper** ([apify/puppeteer-scraper](https://apify.com/apify/puppeteer-scraper))), you can use their built-in `context.saveSnapshot()` function. Once called, it saves a screenshot and HTML into the run's **key-value store**.
 
-When **building your own actors** with [Puppeteer](https://pptr.dev) or the the [Apify SDK](https://sdk.apify.com) package, you can use the powerful `utils.puppeteer.saveSnapshot()` [function](https://sdk.apify.com/docs/api/puppeteer#puppeteersavesnapshot). It allows you name the screenshot, so you can identify it later.
+When **building your own actors** with [Puppeteer](https://pptr.dev) or the [Apify SDK](https://sdk.apify.com) package, you can use the powerful `utils.puppeteer.saveSnapshot()` [function](https://sdk.apify.com/docs/api/puppeteer#puppeteersavesnapshot). It allows you name the screenshot, so you can identify it later.
 
 [Cheerio](https://cheerio.js.org)-based actors do not have a helper function because they allow taking snapshots with a single line of code. Just save the HTML with the correct content type.
 
 ```javascript
-await Apify.setValue('SNAPSHOT', html, { contentType: 'text/html' });
+import { Actor } from 'apify';
+
+// ...
+await Actor.setValue('SNAPSHOT', html, { contentType: 'text/html' });
 ```
 
 #### [](#when-to-save-snapshots) When to save snapshots
 
 The most common approach is to save on error. We can enhance our previous try/catch block like this:
 
 ```javascript
+import { Actor } from 'apify';
+import { puppeteerUtils } from 'crawlee';
+
+// ...
 // storeId is ID of current key value store, where we save snapshots
-const storeId = Apify.getEnv().defaultKeyValueStoreId;
+const storeId = Actor.getEnv().defaultKeyValueStoreId;
```

Comment:
Does get env still exist or is it preferred? I thought the preferred way to read this is now through Configuration.

Reply:
It exists, yeah, but I was not updating much logic, I just replaced the existing wrong usage of `Apify` class - properly updating all examples will take more time, I thought we're focusing on correctness of the existing stuff for now.